custom-error-storage.tf

```diff
@@ -0,0 +1,43 @@
+resource "azurerm_storage_account" "custom_error" {
+  for_each = { for k, v in local.waf_targets : k => v if v["custom_errors"] != null }
+
+  name                          = "${replace("${local.resource_prefix}${each.key}", "-", "")}customerror"
+  resource_group_name           = local.resource_prefix
+  location                      = local.azure_location
+  account_tier                  = "Standard"
+  account_replication_type      = "LRS"
+  min_tls_version               = "TLS1_2"
+  enable_https_traffic_only     = true
+  public_network_access_enabled = true
+
+  static_website {}
+
+  tags = local.tags
+}
+
+resource "azurerm_storage_container" "custom_error_web" {
```

Comment:
The $web container is automatically provisioned by Azure so we do not need to deploy it